VideoLocker/res/layout/row_discussion_comments_comment.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Note that this layout name uses a different convention than `discussion_responses_response_row` where `_row` is a suffix instead of a prefix. Most of the other layouts use this convention as well, including your newly added `discussion_user_profile_row`, but excluding `row_discussion_topic`, `row_discussion_thread`, and `row_discussion_thread_dropdown. We should ideally be using one naming convention for all of these. I'm in favor of using it as a prefix everywhere, or maybe event a secondary prefix after `discussion_` (i.e. `discussion_row_*`), as that makes it easier to narrow down duplicates quickly while searching.